Classes/Filter/EventStreamFilter.php

```diff
@@ -0,0 +1,235 @@
+<?php
+namespace Neos\EventStore\Filter;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Annotations as Flow;
+
+/**
+ * EventStreamFilter
+ */
+class EventStreamFilter
+{
+    /**
+     * @var int
+     */
+    public $minimumVersion = null;
+
+    /**
+     * @var int
+     */
+    public $maximumVersion = null;
+
+    /**
+     * @var \DateTimeImmutable
+     */
+    public $since = null;
+
+    /**
+     * @var \DateTimeImmutable
+     */
+    public $until = null;
+
+    /**
+     * @var string
+     */
+    public $streamName = null;
+
+    /**
+     * @var string
+     */
+    public $boundedContext = null;
```

Comment:
The BC is part of the stream name and IMO should not be part of the ES naming just like "aggregate"